src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -179,10 +179,12 @@ export const View = (props: Props) => {
       dataBreachUnresolvedNum,
       dataBrokerTotalNum,
       dataBrokerAutoFixedNum,
+      dataBrokerManuallyFixedNum,
```

Comment:
nit: should be consolidate the naming so it's not `manually fixed` and `manually resolved`.. maybe just stick with `manually resolved` since it's less likely to be confused with `fixed`

Reply:
080a5120e